VaticanBAV/et/BAVet117.xml

```diff
@@ -562,267 +541,93 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                           <note>
                             The miracle is incomplete and ends abruptly.
                           </note>
-                          <listBibl type="editions">
-                             <bibl>
-                                <ptr target="bm:Budge1900Miracles"/>
-                                <citedRange unit="page">70-72</citedRange>
-                             </bibl>
-                          </listBibl>
-                          <listBibl type="translation" xml:lang="en">
-                             <bibl>
-                                <ptr target="bm:Budge1900Miracles"/>
-                                <citedRange unit="page">131-135</citedRange>
-                             </bibl>
-                          </listBibl>
                         </msItem>
 
                       </msItem>
+                      </msContents>
+                    </msPart>
```

Comment:
the part still needs a physical description, you know at least the extent (1-42) and the number of quires (4)

Reply:
"at least the extent (1-42) and the number of quires (4)" - i meant the `<extent>` element

as in 

```
<extent>Made of <measure unit="leaf">42</measure> folios, in <measure unit="quire">4</measure> quires (I-IV). 
                        </extent>
```